database/seeds/BodySeeder.php

```diff
@@ -17,67 +17,364 @@ class BodySeeder extends Seeder
      */
     public function run()
     {
-        Address::create([
-            'country_id'    =>  21,
-            'street'        =>  'Notelaarsstraat 55',
-            'zipcode'       =>  '1000',
-            'city'          =>  'Brussels',
-        ]);
-        Address::create([
-            'country_id'    =>  153,
-            'street'        =>  'Oude Markt 24',
-            'zipcode'       =>  '7511 GB',
-            'city'          =>  'Enschede',
-        ]);
-        Address::create([
-            'country_id'    =>  81,
-            'street'        =>  'Bergstraße 66c',
-            'zipcode'       =>  '01069',
-            'city'          =>  'Dresden',
+	    BodyType::create([
+            'name'          =>  'antenna', //1
         ]);
 
         BodyType::create([
-            'name'          =>  'local',
+            'name'          =>  'contact antenna', //2
         ]);
 
         BodyType::create([
-            'name'          =>  'special',
+            'name'          =>  'contact', //3
         ]);
 
-        BodyType::create([
-            'name'          =>  'commission-european',
+	    BodyType::create([
+	        'name'	    =>  'commission', //4
```

Comment:
Please note that there is a difference between commissions and committees (not entirely sure about which is which myself)